telemetry/definitions/commonDefinitions.json

```diff
@@ -1078,14 +1078,24 @@
             ]
         },
         {
-            "name": "codeTransformMavenBuildCommand",
+            "name": "codeTransformCommandExecutables",
             "type": "string",
             "description": "Type of maven command",
             "allowedValues": [
                 "mvn",
                 "mvnw",
                 "IDEBundledMaven"
             ]
+        },
+        {
+            "name": "codeTransformCommandNames",
+            "type": "string",
+            "description": "Type of maven, gradle and other command line arguments run in the IDE",
+            "allowedValues": [
```

Comment:
I do agree that we need something like this for the shell commands

Reply:
Lets exhaust all the options we want in here offline.